ci/scripts/bitnami-utils.sh

```diff
@@ -856,6 +856,9 @@ crash_and_restart_postgres() {
   if [[ "$pid" != "" ]]; then
     kill -9 $pid
   fi
-
+  
+  # Be sure that postgres is stopped before starting again
+  pg_ctl stop -D $POSTGRESQL_DATA_DIR || true
```

Comment:
does this ever fail? is `||true` necessary?

Reply:
I will leave this one, as this function will be used in `crash_and_restart_postgres_master` and `crash_and_restart_postgres_replica` functions, which are killing process with `kill -9` signal, so after that I am stopping postgres with `pg_ctl` just in case to be sure that postgres is 100% stopped before starting it again. If it won't be fully stopped we will get error on start and the incoming test case will fail to connect to db